turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -510,7 +510,9 @@ impl BroadcastRun for StandardBroadcastRun {
 fn should_use_merkle_variant(slot: Slot, cluster_type: ClusterType, shred_version: u16) -> bool {
     match cluster_type {
         ClusterType::Testnet => shred_version == 28353,
```

Comment:
While we're in here - should we drop the shred version equality check ? I think as part of upgrade process to get mnb to v1.16, we may do a cluster restart that would involve hard fork which would yield new shred verison:

https://github.com/solana-labs/solana/wiki/v1.16-Release-Schedule

Reply:
done.